requirements.txt

```
@@ -0,0 +1,2 @@
+django==5.0.6
```

Use title `D` letter like in lib name

```suggestion
Django==5.0.6
```